atomic_reactor/constants.py

```diff
@@ -14,6 +14,46 @@
 BUILD_JSON_ENV = 'BUILD_JSON'
 RESULTS_JSON = 'results.json'
 
+JSON_SCHEMA = {
+    "type": "object",
+    "properties": {
+        "image": {"type": "string"},
+        "git_url": {"type": "string"},
+        "source": {
+            "type": "object",
+            "properties": {
+                "provider": {"type": "string"},
+                "url": {"type": "string"}
+            },
+            "required": ["provider"]
+        },
+        "prebuild_plugins": {
+            "type": "array",
+            "items": {
+                "type": "object",
+                "properties": {
+                    "name": {"type": "string"},
+                    "is_allowed_to_fail": {"type": "boolean"},
+                    "args": {"type": "object"},
+                }
+            }
+        },
+        "postbuild_plugins": {
+            "type": "array",
+            "items": {
+                "type": "object",
+                "properties": {
+                    "name": {"type": "string"},
+                    "is_allowed_to_fail": {"type": "boolean"},
+                    "args": {"type": "object"},
+                }
+            }
+        },
```

Comment:
Right, current schema is a simple draft ~~so tests would pass~~, any help making it complete is appreciated